### DISCUSSION TITLE
Embedded: Possible to use version band instead of specific .NET Core SDK version?

### DISCUSSION THREAD

**Discussion by kamranayub** [ravendb/ravendb#12753] [Q&A] [answered]
I'm currently still on 4.2 Embedded (for tests) and I had previously installed 110. Today I was upgrading to 116 and that comes with a required .NET Core 3.1.117 runtime requirement. However, with 3.0 releases, .NET no longer allows us to install previous versions if a new one is present and since I had updated VS 2019, I had 3.1.118 already installed. This made it painful to remove/install/re-install both 117 and 118 to get RavenDB working again.

See this for reference: https://github.com/dotnet/sdk/issues/16399

Microsoft suggests using the band, instead of the specific version:

>  Most people when pinning versions pin to a feature band. Patched versions are almost exclusively security or high value fixes and should not have breaking changes.

Does 5.x address this or would this be useful for 4.2.x going forward and 5.x?

**Comment by ppekrol** [comment, marked as answer]
@kamranayub 

You can set the FrameworkVersion to `4.2.x` in ServerOptions. More info here: https://issues.hibernatingrhinos.com/issue/RavenDB-16333#focus=Comments-67-75858.0-0

**Reply by ppekrol** [reply]
I've created a ticket to support additionally syntax like `3.1.17+`. You can track it here: https://issues.hibernatingrhinos.com/issue/RavenDB-17198

**Reply by kamranayub** [reply]
Thanks, that should do the trick!
